src/python/pants/backend/docker/goals/package_image.py

```diff
@@ -336,18 +338,26 @@ def get_build_options(
             yield from target[field_type].options()
         elif issubclass(field_type, DockerBuildOptionFlagFieldMixin):
             yield from target[field_type].options()
-        elif issubclass(field_type, DockerImageBuildImageCacheToField) or issubclass(
-            field_type, DockerImageBuildImageCacheFromField
+        elif (
+            issubclass(field_type, DockerImageBuildImageCacheToField)
+            or issubclass(field_type, DockerImageBuildImageCacheFromField)
+            or issubclass(field_type, DockerImageBuildImageOutputField)
         ):
```

Comment:
```suggestion
        elif (
            issubclass(field_type, (
              DockerImageBuildImageCacheToField,
              DockerImageBuildImageCacheFromField,
              DockerImageBuildImageOutputField,
            ))
        ):
```

formatting is obv off, but this will be more concise, I think.. ;)

Reply:
or, even better could be (see other comment)
```suggestion
        elif issubclass(field_type, DockerBuildKitOptionField):
```